central/administration/events/datastore/telemetry.go

```diff
@@ -0,0 +1,63 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+)
+
+// Gather the number of administrative events
+func Gather(ds DataStore) phonehome.GatherFunc {
+	return func(ctx context.Context) (map[string]any, error) {
+		ctx = sac.WithGlobalAccessScopeChecker(ctx,
+			sac.AllowFixedScopes(
+				sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+				sac.ResourceScopeKeys(resources.Administration),
+			),
+		)
+		props := map[string]any{}
+		var combinedErr error
+		err := phonehome.AddTotal(ctx, props, "Administrative Events", func(ctx context.Context) (int, error) {
+			return ds.CountEvents(ctx, search.EmptyQuery())
+		})
+		if err != nil {
+			combinedErr = multierror.Append(combinedErr, err)
```

Comment:
Why did you use the `multierror` package over `pkg/errorhelpers`? I would suggest the latter to reduce use of external packages.

Reply:
Tomek suggested multierror and I was unaware we had `pkg/errorhelpers`, I'll switch to using that 👍